static/app/locale.tsx

```diff
@@ -208,9 +209,14 @@ export function parseComponentTemplate(template: string): ParsedTemplate {
       if (closeBraceOrValueSeparator === ']') {
         pos = regex.lastIndex;
       } else {
-        pos = regex.lastIndex = process(regex.lastIndex, groupName, true);
+        pos = regex.lastIndex = process(
+          regex.lastIndex,
+          groupName + groupId.toString(),
+          true
+        );
       }
-      buf.push({group: groupName});
+      buf.push({group: groupName, id: groupName + groupId.toString()});
```

Comment:
nit: we could use only the `id` here 
```suggestion
      buf.push({group: groupName, id: groupId.toString()});
```

Reply:
True, I added the name as it was easier to debug when logging the parsed result.